packages/core/src/types.ts

```diff
@@ -1544,16 +1542,15 @@ export interface ValueAdjacencyMap<
   [stateId: string]: Record<string, State<TContext, TEvent>>;
 }
 
-export interface SCXMLEventMeta<TEvent extends EventObject> {
-  _event: SCXML.Event<TEvent>;
-}
+export type SCXMLEventMeta<TEvent extends EventObject> = StateMeta<any, TEvent>;
```

Comment:
This is somewhat weirdly named - what does it have to do with SCXML now? I'd probably remove this one altogether and rename `StateMeta` to something like `ExecutionMeta`